backend/database/apps.py

```diff
@@ -216,7 +216,8 @@ def record_app_usage(
         timestamp: datetime = None
 ):
     if not memory_id and not message_id:
-        raise ValueError('memory_id or message_id must be provided')
+        if usage_type != UsageHistoryType.transcript_processed_external_integration:
```

Comment:
don't do this, keep the db layer clean please. 

do add a check `memory_id not None` here https://github.com/BasedHardware/omi/pull/1426/files#diff-d8043b840bd855b50c59fe81d36a11f3d7c819da45ecc81870efacb4f89886afR357

keep in mind - every transaction needs a reference(object) id

Reply:
Done